compose/cli/log_printer.py

```diff
@@ -20,6 +22,8 @@ def __init__(self, containers, attach_params=None, output=sys.stdout, monochrome
     def run(self):
         mux = Multiplexer(self.generators)
         for line in mux.loop():
+            if isinstance(line, six.text_type) and not six.PY3:
+                line = line.encode('utf-8')
```

Comment:
Note to self: this is the wrong place for the conversion. The multiplexer should always return bytes in py2 and str in py3.